### DISCUSSION TITLE
Automatic indentation may not reach the appropriate position in a few cases

### DISCUSSION THREAD

**Discussion by lll123github** [microsoft/pylance-release#4760] [Enhancement]
When there is a continue in the previous line, pressing the 'tab' from the beginning of the line will not indent to the appropriate position

Example:

before I press the 'tab'
![image](https://github.com/microsoft/vscode-python/assets/97929788/faaca5d3-98cb-4bff-a422-18f760856602)
after I press  the 'tab'
![image](https://github.com/microsoft/vscode-python/assets/97929788/8290700d-08fa-4af3-92c3-afaa5692d95b)

while before I press the 'enter'
![image](https://github.com/microsoft/vscode-python/assets/97929788/38deca0a-d262-4b8d-bd8a-286ed64be740)
after I press the 'enter'
![image](https://github.com/microsoft/vscode-python/assets/97929788/299f20a5-c648-4100-a7a8-b7ce85b31d95)

**Comment by lll123github** [comment]
Not only 'continue' but 'break' has the same problem

**Comment by judej** [comment]
Pylance does not support auto indentation on `tab`. We could add that functionality based on user demand. Will make this a discussion to look for user feedback for this feature add.
